vars/longevityPipeline.groovy

```diff
@@ -84,12 +83,29 @@ def call(Map pipelineParams) {
                         completed_stages = [:]
                     }
                     dir('scylla-cluster-tests') {
-                        checkout scm
+                        timeout(time: 5, unit: 'MINUTES') {
+                            checkout scm
 
-                        dir("scylla-qa-internal") {
-                            git(url: 'git@github.com:scylladb/scylla-qa-internal.git',
-                                credentialsId:'b8a774da-0e46-4c91-9f74-09caebaea261',
-                                branch: 'master')
+                            dir("scylla-qa-internal") {
+                                git(url: 'git@github.com:scylladb/scylla-qa-internal.git',
+                                    credentialsId:'b8a774da-0e46-4c91-9f74-09caebaea261',
+                                    branch: 'master')
+                            }
+                        }
```

Comment:
Can it be defined as a function too?

Reply:
why no define the timeouts it as stage options ?
https://github.com/scylladb/siren-tests/blob/master/Jenkinsfile#L62

I think it's cleaner then adding one more layer to the tree